gulpfile.ts

```diff
@@ -3,39 +3,41 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// tslint:disable:no-console
-// tslint:disable:no-implicit-dependencies (this allows the use of dev dependencies)
-
-// Grandfathered in
-// tslint:disable:typedef
-// tslint:disable:no-unsafe-any
+// tslint:disable:no-console promise-function-async
 
 import * as cp from 'child_process';
+import * as fse from 'fs-extra';
 import * as glob from 'glob';
 import * as gulp from 'gulp';
 import * as decompress from 'gulp-decompress';
 import * as download from 'gulp-download';
 import * as os from 'os';
 import * as path from 'path';
 
-function test() {
-    const env = process.env;
+const env = process.env;
+
+function webpack(mode: string): cp.ChildProcess {
```

Comment:
Reminding in case you forgot to leverage shared gulp tasks here

Reply:
thx.